client/state/notices/middleware.js

```diff
@@ -175,6 +176,7 @@ export const handlers = {
 	[ PUBLICIZE_CONNECTION_UPDATE_FAILURE ]: onPublicizeConnectionUpdateFailure,
 	[ GUIDED_TRANSFER_HOST_DETAILS_SAVE_SUCCESS ]: dispatchSuccess( translate( 'Thanks for confirming those details!' ) ),
 	[ SITE_FRONT_PAGE_SET_FAILURE ]: dispatchError( translate( 'An error occurred while setting the homepage' ) ),
+	[ THEME_TRY_AND_CUSTOMIZE_FAILURE ]: dispatchError( translate( 'An error occured while trying to customize theme' ) ),
```

Comment:
We had a quick chat. Following the rule of thumb of always give an indication to the user on "what can I do now" instead of blunt error messages, let's go with: 

> "Error customizing theme, check with support"